app/js/app-utilities.js

```diff
@@ -1935,13 +1935,14 @@ appUtilities.launchWithModelFile = function() {
   var uri_path = getParameterByName('uri');
   var chiseInstance = appUtilities.getActiveChiseInstance();
   var promptInvalidURIWarning = this.promptInvalidURIWarning;
+  var promptInvalidFileView = this.promptInvalidFileView;
```

Comment:
@ahmetcandiroglu  warning message of `promptInvalidFileView` is  `Incorrect file format!`. You will need to create a new one and set warning message to `Invalid URL!`